packages/flutter/lib/src/material/snack_bar.dart

```diff
@@ -209,7 +218,10 @@ class _SnackBarActionState extends State<SnackBarAction> {
         backgroundColor: resolveBackgroundColor(),
       ),
       onPressed: _haveTriggeredAction ? null : _handlePressed,
-      child: Text(widget.label),
+      child: Text(
```

Comment:
Do you think if it makes sense to directly add the `lableStyle` here? https://github.com/flutter/flutter/blob/165e535474d5005aa97af79e449f86a7743d2aed/packages/flutter/lib/src/material/snack_bar.dart#L207 because this label is a button, we can just let the `ButtonStyle` to handle the text style.